wordpress-objects/Clarkson_Object.php

```diff
@@ -17,6 +17,7 @@ public function __construct( $post ) {
 			$this->_post = $post;
 		}else{
 			trigger_error("Deprecated __construct called with an ID. Use `::get($post)` instead.", E_USER_DEPRECATED);
+            Clarkson_Debug::callStack();
```

Comment:
If we would like to add the debugging to a different plugin then this could become an `apply_filter`?